packages/vaex-core/src/hash_object.cpp

```diff
@@ -13,15 +13,6 @@ typedef SSIZE_T ssize_t;
 
 namespace py = pybind11;
 
-namespace std {
-    template<>
-    struct hash<PyObject*> {
-        size_t operator()(const PyObject *const &o) const {
-            return PyObject_Hash((PyObject*)o);
-        }
-    };
-}
```

Comment:
This was my best guess...

https://github.com/pybind/pybind11/blob/769fd3b889fef6cddb060f2a0be26aee62b4da05/include/pybind11/pytypes.h#L859

https://github.com/ddelange/vaex/actions/runs/3965609112/jobs/6795506653#step:6:2110

Reply:
@maartenbreddels any thoughts on this (incl me updating the pybind11 submodule)?